From e70c06f8df988824df7b569c4944b1fb9d78c748 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Wed, 21 Nov 2018 16:42:49 +0200
Subject: [PATCH 5181/5242] MLK-20328-9: ARM64: dtsi: imx8qm-device: adjust
 the expected format

commit  e1be71bc77099bcd994c07e052e797800b96b48d from
https://source.codeaurora.org/external/imx/linux-imx.git

Default value must be prefixed by "0".

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
(cherry picked from commit e1f214bd631bda58ab7850866447a53e4a479ac8)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index 6913949..3cea305 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -3884,7 +3884,7 @@
 		clock-names = "bus", "mclk0", "mclk1", "mclk2", "mclk3";
 		dma-names = "rx";
 		dmas = <&edma2 18 0 1>;
-		fsl,dataline = <0xf 0x0>;
+		fsl,dataline = <0 0xf 0x0>;
 		status = "disabled";
 		power-domains = <&pd_sai4>;
 	};
@@ -3901,7 +3901,7 @@
 		clock-names = "bus", "mclk0", "mclk1", "mclk2", "mclk3";
 		dma-names = "tx";
 		dmas = <&edma2 19 0 0>;
-		fsl,dataline = <0x0 0xf>;
+		fsl,dataline = <0 0x0 0xf>;
 		status = "disabled";
 		power-domains = <&pd_sai5>;
 	};
-- 
1.7.9.5

